content/html/reference/elements/input-button/docs.md

```diff
@@ -0,0 +1,48 @@
+---
```

Comment:
Would you mind if I did this as a separate PR? There's at least one place this will need an update in the build-json code, so it seems cleaner to make it its own thing.

Reply:
Oh, sorry, for some reason I thought we could build with distinct names already. This doesn't need to change right now, if that's not the case.